exercise-src/react-native/20-building/01-solution/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "PlaceMyOrder",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "api": "npm run linkLocalhost:android && place-my-order-api --port 7070",
+    "depcheck": "depcheck .",
+    "devtools": "react-devtools",
+    "clean": "rm -rf tsconfig.tsbuildinfo coverage android/.gradle android/build android/app/build node_modules/.cache",
+    "eslint": "eslint *.ts *.js src",
+    "eslint:fix": "eslint --fix *.ts *.js src",
+    "ios": "react-native run-ios",
+    "linkLocalhost:android": "adb reverse tcp:7070 tcp:7070",
+    "precheck": "npm run typecheck && npm run eslint && npm run prettier && npm run depcheck && npm run test",
+    "prettier": "prettier --check *.ts *.js *.json src",
+    "prettier:fix": "prettier --write *.ts *.js *.json src",
+    "start": "react-native start --reset-cache --experimental-debugger",
+    "test": "jest",
+    "test:inspect": "node --inspect-brk ./node_modules/.bin/jest --watch",
+    "test:watch": "jest --watch",
+    "typecheck": "tsc --noEmit",
+    "build": "react-native build-android --mode=release"
```

Comment:
It looks like everything else is in alphabetical order (yay), so let’s put this one in the right place.

Reply:
Also, if this is specific to android, we should probably make it `build:android` instead.